openshift-environment-driver/src/main/java/org/jboss/pnc/environment/openshift/Configurations.java

```diff
@@ -45,11 +46,11 @@
 
     private final String filePath;
 
+    @Inject
```

Comment:
Basically Inject works only in objects instantiated by the CDI. Meaning no Inject when you use new MyClass() or MyClass.fieldOrMethod(), which is kinda the enum case.